### PR TITLE
SW-17536 Add blog post about javascript currency formatting

### PR DESCRIPTION
**Pull request by klarstil**
Please review the spelling and grammar.

### CODE REVIEW

File: source/blog/_posts/2017-01-30-javascript-currency-formatting.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+```
+var toLocaleStringSupportsOptions = function() {
+    return !!(typeof Intl == 'object' && Intl && typeof Intl.NumberFormat == 'function');
+};
+```
+
+### Performance matters
+
+After playing around with both approaches, I noticed a quite heavy downside of `Number.toLocaleString(). It became a
```
Comment (mitelg) [inline source/blog/_posts/2017-01-30-javascript-currency-formatting.md]: a backtick is missing here

File: source/blog/_posts/2017-01-30-javascript-currency-formatting.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+tags:
+- javascript
+- currency formatting
+- native api
+
+categories:
+- dev
+
+authors: [stp]
+github_link: blog/_posts/2017-01-30-javascript-currency-formatting.md
```
Comment (mitelg) [inline source/blog/_posts/2017-01-30-javascript-currency-formatting.md]: maybe the date must be changed?

Comment (klarstil) [inline source/blog/_posts/2017-01-30-javascript-currency-formatting.md]: Done 👍

File: source/blog/_posts/2017-01-30-javascript-currency-formatting.md (hunk 1 above)
Comment (klarstil) [inline source/blog/_posts/2017-01-30-javascript-currency-formatting.md]: Done 👍

**Pull request merged**